doc/EnvironmentVariables.md

```diff
@@ -84,7 +84,10 @@
 | UseServerHeartbeat                      | Sets the client-side heartbeat interval to 60sec for upstream AMQP connections                                                                     | bool                                                                  | true                          |
 | UsePersistentStorage                    | If set, will enable directly persisting messages / twins into the local database before forwarding/processing                                      | bool                                                                  | true                          |
 | EnableNonPersistentStorageBackup        | If messages / twins are not directly persisted, this will backup twins and any unprocessed messages on shutdown so Edge can resume on next startup | bool                                                                  | false                         |
-| MqttBroker__MaxInflightMessages         | Maximum number of QoS 1 messages that can be in the process of being sent simultaneously for a single session                                      | int32                                                                 | 16                         |
-| MqttBroker__MaxQueuedMessages           | Maximum number of messages that can be in the outgoing queue for a single session                                                                  | int32                                                                 | 1000                         |
-| MqttBroker__MaxQueuedBytes              | Maximum number of bytes that can be in the outgoing queue for a single session                                                                     | int32 (arm), int64 (amd64)                                            | 0 (no limit)                         |
-| MqttBroker__WhenFull                    | Defines what to do in case either max queued messages or max queued bytes is reached                                                               | drop_new, drop_old                                                    | drop_new                       |
\ No newline at end of file
+| MqttBroker__MaxInflightMessages         | Maximum number of QoS 1 messages that can be in the process of being sent simultaneously for a single session                                      | int32                                                                 | 16                            |
```

Comment:
Can we do smaller than int32? Seems like it allows for a very large possible set of messages in flight :)

Reply:
We probably should. But let's do that in a separate PR. I haven't touched this setting in the current PR (just whitespace change in the docs)